docs/specifications/tests/Consistency-TP/consistency05.md

```diff
@@ -1,49 +1,205 @@
+            on the public DNS.
+            * The lookup must take into account changes that
+              undelegated data has created, if any.
+         4. If the lookup returns the relevant address record or records
+            with the same owner name as in the query, then extract those 
+            and add to *Address Records From Child* with name and IP 
+            address or addresses.
+      4. Or, if the response has the AA flag unset, then
+         emit *[CHILD_NS_FAILED]*. 
+      5. Or, if the RCODE of the response is neither NOERROR nor 
+         NXDOMAIN, then emit *[CHILD_NS_FAILED]*.
+      6. Or, if the RCODE is NOERROR (with the AA flag set), then
+         extract any addresses records (A, AAAA) from the answer
+         section response whose owner name matches the owner name 
+         of the of the query and add that or those to 
+         *Address Records From Child* with name and IP.
+      7. Or, if the RCODE is NXDOMAIN, there is nothing to do.
+
+   3. If all servers emitted *[NO_RESPONSE]* or *[CHILD_NS_FAILED]*, 
+      then emit *[CHILD_ZONE_LAME]* and completely stop processing 
+      this test case.
+
+6. Compare the IP address for the name servers from 
+   *Delegation Strict Glue* with *Address Records From Child*
+   (i.e. [in-bailiwick] only).
+
+   1. If an IP from *Delegation Strict Glue* is not listed in 
+      *Address Records From Child* with that same name server name 
+      then emit *[IN_BAILIWICK_ADDR_MISMATCH]*.
+
+   2. If an IP from *Address Records From Child* is not listed in
+      *Delegation Strict Glue* with that same name server name then 
+      emit *[EXTRA_ADDRESS_CHILD]*.
+
+7. For each name server name in *Delegation Extended Glue* 
+   (i.e. [out-of-bailiwick] only) do: 
+
+   1. Do two DNS lookups, one record type A and one record type 
+      AAAA for name server name as owner name on public DNS.
+
+   2. For each IP address for name server name in in 
+      *Delegation Extended Glue* do:
+      1. If the address is not listed with the same owner name in 
+         the responses of the DNS lookups, or if there was no
+         response, emit *[OUT_OF_BAILIWICK_ADDR_MISMATCH]*.
+
+8. If none of the messages *[IN_BAILIWICK_ADDR_MISMATCH]*, 
+   *[EXTRA_ADDRESS_CHILD]* or *[OUT_OF_BAILIWICK_ADDR_MISMATCH]* has 
+   been emitted, emit *[ADDRESSES_MATCH]*.
+
+
+## Outcome(s)
+
+The outcome of this Test Case is "fail" if there is at least one message
+with the severity level *ERROR* or *CRITICAL*.
+
+The outcome of this Test Case is "warning" if there is at least one message
+with the severity level *WARNING*, but no message with severity level
+*ERROR* or *CRITICAL*.
+
+In other cases the outcome of this Test Case is "pass".
+
+Message                           | Default severity level (if message is emitted)
+:---------------------------------|:-----------------------------------
+CHILD_NS_FAILED                   | NOTICE
+NO_RESPONSE                       | NOTICE
```

Comment:
The Zonemaster policy has the severity of CONSISTENCY:NO_RESPONSE be WARNING for the other test cases. Here it shows up as NOTICE. The severity could be changed to WARNIING here, or the name of the message could be changed.